README.md

```diff
@@ -7,11 +7,19 @@ This project provides a trustless ability to buy an ERC-20 token in exchange for
 ## Tests
 
 Since we're running some tests with a mainnet fork, to get the best trace include your Etherscan API key when running tests:
+
 ```sh
 forge test --etherscan-api-key <your api key here>
 ```
 
 If you're ok with a less-readable trace simply run:
+
 ```sh
 forge test
-```
\ No newline at end of file
+```
+
+## Deploy to Rinkeby
+
+```sh
+forge script script/Deploy.s.sol --broadcast --verify -vvvv --chain-id 4 --private-key <paste private key here> --rpc-url <rinkeby RPC URL here>
```

Comment:
FYI, I like to use `--keystores` instead of pasting private keys

Reply:
cool, updated readme